types/ember__template/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for non-npm package @ember/template 3.0
+// Project: https://emberjs.com/api/ember/3.12/modules/@ember%2Ftemplate
+// Definitions by: Mike North <https://github.com/mike-north>
+//                 Scott Kidder <https://github.com/scottkidder>
```

Comment:
While anyone can add themselves to these headers in DT, you should be aware of what this means
- your own code changes will be merged in automatically _without a review_, and _in 24 hours, instead of the usual 5-7 days_
- your approvals to incoming PRs will cause them to automatically be merged, and then new releases of these packages will be immediately cut and released.

If things go wrong as a result of your code changes and/or approvals, are you willing to dig in on short notice and untangle things so we can unblock the community?

Reply:
Hi Mike, given that I'm still novice with TypeScript I think it's better that I remove myself from the headers. Thanks for the information!

Reply:
@mike-north should we add the Ember TS team so you're not solely responsible?

Reply:
@jamescdavis that sounds good to me. Mind opening a separate PR to do this?

Reply:
Happy to once these get merged.